rnd/docker-compose.yml

```diff
@@ -16,6 +15,32 @@ services:
     networks:
       - app-network
 
+  server_base:
+    build:
+      context: ../
+      dockerfile: rnd/autogpt_server/Dockerfile
+      target: server
+    image: autogpt_server:latest
+    command: ["echo", "This is a base image and should not be run directly"]
+    
+  migrate:
+    image: autogpt_server:latest
+    command: ["sh", "-c", "until pg_isready -h postgres -U agpt_user -d agpt_local; do echo 'Waiting for postgres...'; sleep 2; done; poetry run prisma db push"]
```

Comment:
Q; how is this different to service healthy condition?

Reply:
It shouldn't be, yet for some reason the container seems to run before Postgres is ready even with the healthy condition